datastore/gcloud/aio/datastore/value.py

```diff
@@ -50,9 +50,8 @@ def from_repr(cls, data: Dict[str, Any]) -> 'Value':
         else:
             supported = [name.value for name in supported_types.values()]
             raise NotImplementedError(
-                '{vals} does not contain a supported value type'
-                ' (any of: {supported})'.format(vals=data.keys(),
-                                                supported=supported))
+                f'{data.keys()} does not contain a supported value type '
+                f'(any of: {supported})')
```

Comment:
...huh, that's a great point. Well, not for gcloud-rest: [the conversion script already handles that](https://github.com/talkiq/gcloud-aio/blob/a85d5dc85cbabbd4737bb5f8832c6f9b2a3e6a89/bin/build-rest#L47), but shouldn't AIO still support 3.6?

I have a feeling we've been breaking py36 for a long time, since there's a whole bunch of f-strings on master already. I'm thinking we might need a follow-up to investigate that.

Reply:
Ok, I got myself super confused here. @caseydialpad and I misunderstood Jon's comment here as saying f-strings are for >=3.7, but f-strings were introduced and supported for >=3.6. Our conversion script handes removing the f-strings for gcloud-rest via my link above.

So all is well, nothing to see here!

Reply:
Oh, and suddenly I realize what you mean. Looks like we're [explicitly not fixing f-strings in this file](https://github.com/talkiq/gcloud-aio/blob/a85d5dc85cbabbd4737bb5f8832c6f9b2a3e6a89/bin/build-rest#L29). Lets see how easily fixable that is...

Reply:
Ok, so removing that denylist and rebuilding shows:
```
~/s/t/gcloud-aio » diff -r oldbuild build
diff -r oldbuild/datastore/gcloud/rest/datastore/value.py build/datastore/gcloud/rest/datastore/value.py
53,54c53,54
<                 f'{data.keys()} does not contain a supported value type '
<                 f'(any of: {supported})')
---
>                 '{} does not contain a supported value type '
>                 '(any of: {})'.format((data.keys()), (supported)))
86,87c86,87
<             raise NotImplementedError(f'{kind} is not a supported value type '
<                                       f'(any of: {supported_types})') from e
---
>             raise NotImplementedError('{} is not a supported value type '
>                                       '(any of: {})'.format((kind), (supported_types))) from e
```

So it seems as though we should be fine to use f-strings and we no longer need to avoid properly formatting those files for gcloud-rest. Not sure why we originally needed to do this, but it certainly seems fine now.